File: ClientApp/src/app/model/orderSummaryVm.ts
import { User } from "oidc-client";
import { OneUserOrderVm } from "./oneUserOrderVm";

export class OrderSummaryVm {
    
    public Id: string;
    public PaymentUrl: string;
    public Date: Date;
    public OneUserOrders: OneUserOrderVm[];
    
    // public checkbox: boolean;
}

File: Model/ViewModel/OrderSummaryVm.cs
using System;
using System.Collections.Generic;
using targeteo.pl.Model.Entities;

namespace targeteo.pl.Model.ViewModel
{
    public class OrderSummaryVm
    {
        public string Id { get; set; }
        public string PaymentUrl { get; set; }
        public DateTime Date { get; set; }

        public ICollection<OneUserOrderVM> OneUserOrders { get; set; }
    }
}
